tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     "input": [
+      "hcg7_center.dec"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra.hour"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now that we have a `SkyCoord` object, we can start using it to access data from the [Sloan Digitial Sky Survey](http://www.sdss.org/).  Lets start by just trying to get a picture using the SDSS image cutout service."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "impix = 1024\n",
+      "imsize = 12*u.arcmin\n",
+      "cutoutbaseurl = 'http://skyservice.pha.jhu.edu/DR12/ImgCutout/getjpeg.aspx'\n",
+      "cutouturl = cutoutbaseurl + '?' + urllib.urlencode(dict(ra=hcg7_center.ra.deg, dec=hcg7_center.dec.deg, \n",
+      "                                                        width=impix, height=impix, \n",
+      "                                                        scale=imsize.to(u.arcsec).value/impix))\n",
+      "urllib.urlretrieve(cutouturl, 'HCG7_SDSS_cutout.jpg')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This requires an internet connection, but if it failed don't worry: the file is included in the repository so you can just pretend it suceeded.  Now lets take a look at the image."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "IPython.display.Image('HCG7_SDSS_cutout.jpg')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Very pretty!  This shows HCG7 is in the SDSS imaging survey, so we can use the cells below to download catalogs of objects directly from the [Sloan Digitial Sky Survey](http://www.sdss.org/) SQL database using the [astroquery](https://astroquery.readthedocs.org) affiliated package. This will require an internet connection and a working install of astroquery (If you don't have these you can just skip down two cells, because the data files are provided with the repository). Depending on your version of astroquery it might also issue a warning, which you should be able to safely ignore."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astroquery.sdss import SDSS\n",
+      "sdss = SDSS.query_region(coordinates=hcg7_center, radius=20*u.arcmin, spectro=True, photoobj_fields=['ra','dec','u','g','r','i','z'])"
```

Comment:
I dunno, the way I see it you're translating from one convention for separating words in a multi-word variable name.  In one case it's CamelCase, while in other convention it's underscore_separated.  What's important to the spelling is not where the capitalization or punctuation lies so much as how separations between words are indicated in a particular convention.

I think in cases like this a mixed convention, while sort of ugly looking, is also fine.